guard/README.md

```diff
@@ -265,6 +265,38 @@ You can write a YAML-formatted unit test file (test.yml) as:
 You can then test your rules file using the `test` command as:
 
 ```bash
-$ cfn-guard test -r rules.guard -t test.yml 
+$ cfn-guard test -r rules.guard -t test.yml
 PASS Expected Rule = assert_all_resources_have_non_empty_tags, Status = SKIP, Got Status = SKIP
 PASS Expected Rule = assert_all_resources_have_non_empty_tags, Status = FAIL, Got Status = FAIL
+```
+
+### Autocompletions
+
+To setup Autocompletions you will need to follow instructions for the specific shell your are running.
+
+Currently guard only supports autocompletions for zsh, bash, and fish shells. If you would like autocompletions for a specific shell feel free to open up a new github issue.
+
+Autocompletions are only something available for version >= 3.0
+
+#### zsh
+
+```sh
+    cfn-guard --completions --shell='zsh' > usr/local/share/zsh/site-functions/_cfn-guard && compinnit
+```
+
+#### bash
+
+```bash
+    cfn-guard --completions --shell='bash' > ~/cfn-guard.bash && source ~/cfn-guard.bash
+```
+
+#### fish
+
+```sh
+    cfn-guard --completions --shell='fish' > ~/cfn-guard.fish
+    cd ~
+    ./ ./cfn-guard.fish
```

Comment:
What does the last line of command supposed to do? Source the script we just created? 

Should it be `. ./cfn-guard.fish` or `./cfn-guard.fish`? 

Also, do you think we should start with a folder named `~/.cfn-guard` and then create a script called `autocompletions.sh` in it for future extensibility?

Reply:
Im not 100% sure what the last line does exactly, since ive only ever used fish shell to test this out. Im assuming its similar to what we do in bash and zsh where we source the script

i think its best to leave the user to make their decisions on where they want to export the script.